.github/workflows/build-push-gcr.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Push monolith to Google Cloud
+
+on:
+  push:
+    branches: [master]
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  BRANCH: ${{ github.ref }}
+
+jobs:
+  build-and-push-gcr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Configure gCloud credentials
+        id: gcloud-login
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

Comment:
Should this be tied to a version or a hash?

Reply:
good catch